src/main/scala/com/databricks/labs/overwatch/env/Database.scala

```diff
@@ -155,9 +155,9 @@ class Database(config: Config) extends SparkSessionWrapper {
          |""".stripMargin)
     df.write.format("delta").save(dfTempPath)
 
-    spark.conf.set("spark.sql.files.maxPartitionBytes", 1024 * 1024 * 16) // maximize parallelism on re-read and let
+    // maximize parallelism on re-read and let AQE bring it back down
+    spark.conf.set("spark.sql.files.maxPartitionBytes", 1024 * 1024 * 16)
```

Comment:
why 16 mb ?

Reply:
I didn't change this. This is to increase parallelism on the persistAndLoad functions to get more parallelism during load and allow AQE to coalesce the partitions during the subsequent shuffles.